post_office/models.py

```diff
@@ -242,6 +242,7 @@ class Attachment(models.Model):
     name = models.CharField(_('Name'),max_length=255, help_text=_("The original filename"))
     emails = models.ManyToManyField(Email, related_name='attachments',
                                     verbose_name=_('Email addresses'))
+    mimetype = models.CharField(max_length=255, null=True)
```

Comment:
I think the best practice is to avoid having `None` in `CharField` unless it's really needed to avoid having two representation of empty values, `None` and empty string.

On the Django side, we can simply do `msg.attach(attachment.name, attachment.file.read(), mimetype=attachment.mimetype or None)` so `msg.attach` will receive `None` if `mimetype` is empty.